### PR TITLE
Okazaki - Molecular Test - P1

### PR DESCRIPTION
**Pull request by wwysoc2**

### CODE REVIEW

**Review by ebusygina — APPROVED** [review]
looks good to me

File: src/gdcdictionary/schemas/molecular_test.yaml [hunk 1]
```diff
@@ -7333,6 +7331,12 @@ properties:
       - 76-100%
       - Unknown
       - Not Reported
+      - 0-25
+      - 26-50
+      - 51-75
+      - 76-100
+      - 1-4%
+      - 5-9%
```
Comment (wteouchicago) [inline src/gdcdictionary/schemas/molecular_test.yaml]: do these enums need to be organized (i.e. "sorted") differently, e.g. 1-4% and 5-9% should be before <10%? (or at least before 10-19%? 😵 ) or at least before Unknown and Not Reported?

Comment (wwysoc2) [reply src/gdcdictionary/schemas/molecular_test.yaml]: Oh yes, 1-4, 5-9 should be above Unknown and Not Reported.  I think the other four that were added should be deprecated.  I'll fix.

**Review by wteouchicago — APPROVED** [review]

**Review by stilesj-uchicago — APPROVED** [review]
DICT-523 phase 1 looks good to me

**Pull request merged**